c9s-devel-compose.repo

```diff
@@ -0,0 +1,35 @@
+[baseos-devel]
+name=CentOS Stream 9 development - BaseOS
+baseurl=https://composes.stream.centos.org/development/latest-CentOS-Stream/compose/BaseOS/$basearch/os
+gpgcheck=0
+repo_gpgcheck=0
+enabled=1
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Testing
+priority=1000
```

Comment:
(Note I added this line to ensure these are off by default for now, so we're matching the "production" mirrors)